.github/workflows/unit-test.yml

```diff
@@ -70,10 +77,6 @@ jobs:
                   npm ci
                   npm run build
 
-            - name: Install WordPress
-              run: |
-                  npm run wp-env start
-
```

Comment:
`npm run test:unit:php` already calls `npm run wp-env start`, so this is duplication.